rfhub2/cli/keywords_importer.py

```diff
@@ -346,144 +374,135 @@ def _has_test_case_table(data: str) -> bool:
 
     @staticmethod
     def _get_obsolete_collections_ids(
-        existing_collections: List[Dict], new_collections: List[Dict]
+        existing_collections: List[Collection],
+        new_collections: List[CollectionUpdateWithKeywords],
     ) -> Set[int]:
         """Returns set of collection ids that were found in application but not in paths"""
         new_collections_paths = {
-            new_collection["path"] for new_collection in new_collections
+            new_collection[0].path for new_collection in new_collections
```

Comment:
Type annotation for new collections does not match actual type. There is CollectionUpdateWithKeywords dataclass in annotation and the code is using a tuple.